src/Meilisearch/MeilisearchClient.cs

```diff
@@ -189,5 +190,17 @@ public async Task<DumpStatus> GetDumpStatus(string uid)
 
             return await response.Content.ReadFromJsonAsync<DumpStatus>();
         }
+
+        /// <summary>
+        /// Deletes the index.
+        /// It's not a recovery delete. You will also lose the documents within the index.
+        /// </summary>
+        /// <param name="uid">unique dump identifier.</param>
+        /// <returns>Returns the status of delete operation.</returns>
+        public async Task<bool> DeleteIndex(string uid)
+        {
+            var responseMessage = await this.client.DeleteAsync($"/indexes/{uid}");
```

Comment:
To avoid code duplication, can you use the `Delete()` method in `Index` instead?

Like:
```c#
return await this.Index(uid).Delete();
```